docs/topics/multiplatform/native-spm.md

```diff
@@ -1,5 +1,15 @@
 [//]: # (title: Swift package export setup)
 
+<tldr>
+   This is a remote integration method that can work for you if:<br/>
+
+   * You want to separate the code base of your final application from the common code base.
```

Comment:
```suggestion
   * You want to separate the codebase of your final application from the common codebase.
```

As far as I can see there is no one true spelling, but elsewhere on kotlinlang.org we use this as a one word term.